internal/reconcile/state.go

```diff
@@ -131,6 +131,10 @@ func DetermineReleaseState(ctx context.Context, cfg *action.ConfigFactory, req *
 	case helmrelease.StatusUninstalled:
 		return ReleaseState{Status: ReleaseStatusAbsent, Reason: "found uninstalled release in storage"}, nil
 	case helmrelease.StatusDeployed:
+		//verify if postrender digest has changed
+		if req.PreviousPostrendersDigest != req.Object.Status.LastAttemptedPostRenderersDigest {
```

Comment:
This will trigger an upgrade storm for all releases in a cluster since the `.status.lastAttemptedPostRenderersDigest ` is not present in v2beta2.